core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -464,6 +464,11 @@ public boolean namespaceExists(Namespace namespace) {
     return JdbcUtil.namespaceExists(catalogName, connections, namespace);
   }
 
+  @Override
+  protected Map<String, String> properties() {
```

Comment:
I'm confused, because the changes in this file should be gone after a rebase

Reply:
sorry, I didn't build locally.
thanks, I agree it should be gone. Fixed it.